extension/app/ts/components/subcomponents/CopyToClipboard.tsx

```diff
@@ -1,13 +1,13 @@
 import { clipboardCopy } from './clipboardcopy.js'
 
-interface Props {
+interface CopyToClipboardParams {
 	children: preact.VNode
 	content: string
 	contentDisplayOverride?: string
-	copyMessage: string | undefined
+	copyMessage?: string
 }
 
-export function CopyToClipboard(props: Props): preact.VNode {
+export function CopyToClipboard(props: CopyToClipboardParams): preact.VNode {
```

Comment:
The inferred type will likely be better than an explicit type here.
```suggestion
export function CopyToClipboard(props: CopyToClipboardParams) {
```

Generally speaking, it is better to use inferred types over explicit types unless inference isn't doing the right thing for some reason.